commons/src/main/java/org/dhis2/commons/orgunitselector/OUTreeRepository.kt

```diff
@@ -12,7 +12,7 @@ class OUTreeRepository(
         return availableOrgUnits.withParents().sortedBy { it.displayNamePath()?.joinToString(" ") }
     }
 
-    fun childrenOrgUnits(parentUid: String): List<OrganisationUnit> = availableOrgUnits
+    fun childrenOrgUnits(parentUid: String): List<OrganisationUnit> = orgUnitRepositoryConfiguration.orgUnitRepository(null).withParents()
```

Comment:
Why using `null`? could we use the `parentUid` to get the list from the repository?

Reply:
yes maybe I could refactor the naming for all this, if you review the code you will see that it is using the parentUid later on, I updated the availableOrgUnits call to avoid possible cached values like has happened in other occassions.

Reply:
updated!